### ISSUE TITLE
Delighted Connector: 404 on Survey Responses Treated as a Fatal Error

### ISSUE THREAD

**Issue by RobertKeyser** [ethyca/fides#3145]
### Bug Description

When a user without any data in Delighted submits a subject access request, Fides errors the request, because it encounters a 404 on the Survey Responses endpoint.

The Fides Delighted connector treats `404`s encountered on the https://api.delighted.com/v1/survey_responses.json endpoint as fatal, but they appear to be just caused by the lack of survey responses for a given email address.

<details><summary>Click to view full stack trace</summary>

```
2023-04-24 16:18:43 2023-04-24 21:18:43.581 [INFO] (saas_connector:create_client:151): Creating client to https://api.delighted.com
2023-04-24 16:18:43 2023-04-24 21:18:43.582 [DEBUG] (cache:get_cache:178): Testing Redis connection...
2023-04-24 16:18:43 2023-04-24 21:18:43.582 [DEBUG] (cache:get_cache:184): Redis connection succeeded.
2023-04-24 16:18:43 2023-04-24 21:18:43.891 [INFO] (authenticated_client:log_request_and_response_for_debugging:213): 
2023-04-24 16:18:43 
2023-04-24 16:18:43 -----------SAAS REQUEST-----------
2023-04-24 16:18:43 GET https://api.delighted.com/v1/people.json?email=[REDACTED]%40ethyca.com
2023-04-24 16:18:43 headers: {'Authorization': 'Basic [REDACTED]'}
2023-04-24 16:18:43 body: None
2023-04-24 16:18:43 response: b'[]'
2023-04-24 16:18:43 2023-04-24 21:18:43.892 [INFO] (saas_connector:execute_prepared_request:290): 0 row(s) returned after postprocessing 'person' collection.
2023-04-24 16:18:43 2023-04-24 21:18:43.894 [INFO] (graph_task:pre_process_input_data:322): Consolidating incoming data into delighted:person from __ROOT__:__ROOT__.
2023-04-24 16:18:43 2023-04-24 21:18:43.896 [INFO] (graph_task:log_end:422): Ending pri_5666aa57-dcec-4124-88f4-83fa8f8522fa, delighted:person
2023-04-24 16:18:43 2023-04-24 21:18:43.920 [INFO] (graph_task:log_start:379): Starting pri_5666aa57-dcec-4124-88f4-83fa8f8522fa, traversal_node delighted:survey_response
2023-04-24 16:18:43 2023-04-24 21:18:43.923 [INFO] (graph_task:pre_process_input_data:322): Consolidating incoming data into delighted:survey_response from __ROOT__:__ROOT__.
2023-04-24 16:18:43 2023-04-24 21:18:43.925 [DEBUG] (cache:get_cache:178): Testing Redis connection...
2023-04-24 16:18:43 2023-04-24 21:18:43.925 [DEBUG] (cache:get_cache:184): Redis connection succeeded.
2023-04-24 16:18:43 2023-04-24 21:18:43.926 [INFO] (saas_query_config:get_erasure_request_by_action:108): Unable to find matching endpoint to delete 'survey_response' collection
2023-04-24 16:18:43 2023-04-24 21:18:43.927 [INFO] (saas_query_config:generate_query:306): Populated request params for /v1/survey_responses.json
2023-04-24 16:18:43 2023-04-24 21:18:43.927 [DEBUG] (cache:get_cache:178): Testing Redis connection...
2023-04-24 16:18:43 2023-04-24 21:18:43.927 [DEBUG] (cache:get_cache:184): Redis connection succeeded.
2023-04-24 16:18:43 2023-04-24 21:18:43.928 [INFO] (saas_connector:create_client:151): Creating client to https://api.delighted.com
2023-04-24 16:18:43 2023-04-24 21:18:43.928 [DEBUG] (cache:get_cache:178): Testing Redis connection...
2023-04-24 16:18:43 2023-04-24 21:18:43.928 [DEBUG] (cache:get_cache:184): Redis connection succeeded.
2023-04-24 16:18:44 2023-04-24 21:18:44.223 [INFO] (authenticated_client:log_request_and_response_for_debugging:213): 
2023-04-24 16:18:44 
2023-04-24 16:18:44 -----------SAAS REQUEST-----------
2023-04-24 16:18:44 GET https://api.delighted.com/v1/survey_responses.json?person_email=[REDACTED]%40ethyca.com
2023-04-24 16:18:44 headers: {'Authorization': 'Basic [REDACTED]'}
2023-04-24 16:18:44 body: None
2023-04-24 16:18:44 response: b'{"status":404,"message":"Record not found"}'
2023-04-24 16:18:44 2023-04-24 21:18:44.224 [WARNING] (graph_task:result:136): Retrying access_request delighted:survey_response in 1 seconds...
2023-04-24 16:18:45 2023-04-24 21:18:45.230 [WARNING] (graph_task:log_end:414): Ending pri_5666aa57-dcec-4124-88f4-83fa8f8522fa, delighted:survey_response with failure Client call failed with status code '404'
2023-04-24 16:18:45 2023-04-24 21:18:45.243 [DEBUG] (cache:get_cache:178): Testing Redis connection...
2023-04-24 16:18:45 2023-04-24 21:18:45.244 [DEBUG] (cache:get_cache:184): Redis connection succeeded.
2023-04-24 16:18:45 2023-04-24 21:18:45.262 [DEBUG] (task_resources:close:198): Closing all task resources for pri_5666aa57-dcec-4124-88f4-83fa8f8522fa
2023-04-24 16:18:45 2023-04-24 21:18:45.274 [ERROR] (logger:_log_exception:22): Client call failed with status code '404'
2023-04-24 16:18:45 Traceback (most recent call last):
2023-04-24 16:18:45 
2023-04-24 16:18:45   File "/fides/src/fides/api/ops/util/wrappers.py", line 12, in wrap
2023-04-24 16:18:45     loop = asyncio.get_running_loop()
2023-04-24 16:18:45            │       └ <built-in function get_running_loop>
2023-04-24 16:18:45            └ <module 'asyncio' from '/usr/local/lib/python3.10/asyncio/__init__.py'>
2023-04-24 16:18:45 
2023-04-24 16:18:45 RuntimeError: no running event loop
2023-04-24 16:18:45 
2023-04-24 16:18:45 
2023-04-24 16:18:45 During handling of the above exception, another exception occurred:
2023-04-24 16:18:45 
2023-04-24 16:18:45 
2023-04-24 16:18:45 Traceback (most recent call last):
2023-04-24 16:18:45 
2023-04-24 16:18:45   File "/usr/local/lib/python3.10/threading.py", line 973, in _bootstrap
2023-04-24 16:18:45     self._bootstrap_inner()
2023-04-24 16:18:45     │    └ <function Thread._bootstrap_inner at 0xffff82514dc0>
2023-04-24 16:18:45     └ <WorkerThread(AnyIO worker thread, started 281472007131616)>
2023-04-24 16:18:45   File "/usr/local/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
2023-04-24 16:18:45     self.run()
2023-04-24 16:18:45     │    └ <function WorkerThread.run at 0xffff56b39b40>
2023-04-24 16:18:45     └ <WorkerThread(AnyIO worker thread, started 281472007131616)>
2023-04-24 16:18:45   File "/root/.local/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 867, in run
2023-04-24 16:18:45     result = context.run(func, *args)
2023-04-24 16:18:45              │       │   │      └ ()
2023-04-24 16:18:45              │       │   └ functools.partial(<function create_privacy_request at 0xffff5f40f400>, db=<sqlalchemy.orm.session.ExtendedSession object at 0...
2023-04-24 16:18:45              │       └ <method 'run' of '_contextvars.Context' objects>
2023-04-24 16:18:45              └ <_contextvars.Context object at 0xffff56cd6340>
2023-04-24 16:18:45 
2023-04-24 16:18:45   File "/fides/src/fides/api/ops/api/v1/endpoints/privacy_request_endpoints.py", line 194, in create_privacy_request
2023-04-24 16:18:45     return create_privacy_request_func(db, config_proxy, data, False)
2023-04-24 16:18:45 NoneType: None
2023-04-24 16:18:45            │                           │   │             └ [PrivacyRequestCreate(external_id=None, started_processing_at=None, finished_processing_at=None, requested_at=None, identity=...
2023-04-24 16:18:45            │                           │   └ <fides.core.config.config_proxy.ConfigProxy object at 0xffff545c38b0>
2023-04-24 16:18:45            │                           └ <sqlalchemy.orm.session.ExtendedSession object at 0xffff545c1ff0>
2023-04-24 16:18:45            └ <function create_privacy_request_func at 0xffff5f059480>
2023-04-24 16:18:45 
2023-04-24 16:18:45   File "/fides/src/fides/api/ops/api/v1/endpoints/privacy_request_endpoints.py", line 1698, in create_privacy_request_func
2023-04-24 16:18:45     queue_privacy_request(privacy_request.id)
2023-04-24 16:18:45     │                     │               └ <sqlalchemy.orm.attributes.InstrumentedAttribute object at 0xffff7cb36d90>
2023-04-24 16:18:45     │                     └ <fides.api.ops.models.privacy_request.PrivacyRequest object at 0xffff550570a0>
2023-04-24 16:18:45     └ <function queue_privacy_request at 0xffff5fd6c160>
2023-04-24 16:18:45 
2023-04-24 16:18:45   File "/fides/src/fides/api/ops/service/privacy_request/request_runner_service.py", line 259, in queue_privacy_request
2023-04-24 16:18:45     task = run_privacy_request.delay(
2023-04-24 16:18:45            └ <@task: fides.api.ops.service.privacy_request.request_runner_service.run_privacy_request of fides.api.ops.tasks at 0xffff7cae...
2023-04-24 16:18:45 
2023-04-24 16:18:45   File "/root/.local/lib/python3.10/site-packages/celery/app/task.py", line 425, in delay
2023-04-24 16:18:45     return self.apply_async(args, kwargs)
2023-04-24 16:18:45            │    │           │     └ {'privacy_request_id': 'pri_5666aa57-dcec-4124-88f4-83fa8f8522fa', 'from_webhook_id': None, 'from_step': None}
2023-04-24 16:18:45            │    │           └ ()
2023-04-24 16:18:45            │    └ <function Task.apply_async at 0xffff7cb2e050>
2023-04-24 16:18:45            └ <@task: fides.api.ops.service.privacy_request.request_runner_service.run_privacy_request of fides.api.ops.tasks at 0xffff7cae...
2023-04-24 16:18:45   File "/root/.local/lib/python3.10/site-packages/celery/app/task.py", line 572, in apply_async
2023-04-24 16:18:45     return self.apply(args, kwargs, task_id=task_id or uuid(),
2023-04-24 16:18:45            │    │     │     │               │          └ <function uuid at 0xffff7cd5ae60>
2023-04-24 16:18:45            │    │     │     │               └ None
2023-04-24 16:18:45            │    │     │     └ {'privacy_request_id': 'pri_5666aa57-dcec-4124-88f4-83fa8f8522fa', 'from_webhook_id': None, 'from_step': None}
2023-04-24 16:18:45            │    │     └ []
2023-04-24 16:18:45            │    └ <function Task.apply at 0xffff7cb2e290>
2023-04-24 16:18:45            └ <@task: fides.api.ops.service.privacy_request.request_runner_service.run_privacy_request of fides.api.ops.tasks at 0xffff7cae...
2023-04-24 16:18:45   File "/root/.local/lib/python3.10/site-packages/celery/app/task.py", line 793, in apply
2023-04-24 16:18:45     ret = tracer(task_id, args, kwargs, request)
2023-04-24 16:18:45           │      │        │     │       └ {'id': 'd24b160d-f86a-4f7e-9894-2b131cfa76a7', 'retries': 0, 'is_eager': True, 'logfile': None, 'loglevel': 0, 'hostname': '5...
2023-04-24 16:18:45           │      │        │     └ {'privacy_request_id': 'pri_5666aa57-dcec-4124-88f4-83fa8f8522fa', 'from_webhook_id': None, 'from_step': None}
2023-04-24 16:18:45           │      │        └ ()
2023-04-24 16:18:45           │      └ 'd24b160d-f86a-4f7e-9894-2b131cfa76a7'
2023-04-24 16:18:45           └ <function build_tracer.<locals>.trace_task at 0xffff550ca050>
2023-04-24 16:18:45   File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 451, in trace_task
2023-04-24 16:18:45     R = retval = fun(*args, **kwargs)
2023-04-24 16:18:45                  │    │       └ {'privacy_request_id': 'pri_5666aa57-dcec-4124-88f4-83fa8f8522fa', 'from_webhook_id': None, 'from_step': None}
2023-04-24 16:18:45                  │    └ ()
2023-04-24 16:18:45                  └ <bound method run_privacy_request of <@task: fides.api.ops.service.privacy_request.request_runner_service.run_privacy_request...
2023-04-24 16:18:45 
2023-04-24 16:18:45   File "/fides/src/fides/api/ops/util/wrappers.py", line 15, in wrap
2023-04-24 16:18:45     return asyncio.run(func(*args, **kwargs))
2023-04-24 16:18:45            │       │   │     │       └ {'privacy_request_id': 'pri_5666aa57-dcec-4124-88f4-83fa8f8522fa', 'from_webhook_id': None, 'from_step': None}
2023-04-24 16:18:45            │       │   │     └ (<@task: fides.api.ops.service.privacy_request.request_runner_service.run_privacy_request of fides.api.ops.tasks at 0xffff7ca...
2023-04-24 16:18:45            │       │   └ <function run_privacy_request at 0xffff5fd6c1f0>
2023-04-24 16:18:45            │       └ <function run at 0xffff82329ea0>
2023-04-24 16:18:45            └ <module 'asyncio' from '/usr/local/lib/python3.10/asyncio/__init__.py'>
2023-04-24 16:18:45 
2023-04-24 16:18:45   File "/usr/local/lib/python3.10/asyncio/runners.py", line 44, in run
2023-04-24 16:18:45     return loop.run_until_complete(main)
2023-04-24 16:18:45            │    │                  └ <coroutine object run_privacy_request at 0xffff544719a0>
2023-04-24 16:18:45            │    └ <method 'run_until_complete' of 'uvloop.loop.Loop' objects>
2023-04-24 16:18:45            └ <uvloop.Loop running=True closed=False debug=False>
2023-04-24 16:18:45 
2023-04-24 16:18:45 > File "/fides/src/fides/api/ops/service/privacy_request/request_runner_service.py", line 356, in run_privacy_request
2023-04-24 16:18:45     access_result: Dict[str, List[Row]] = await run_access_request(
2023-04-24 16:18:45                    │         │    │             └ <function run_access_request at 0xffff5fd12290>
2023-04-24 16:18:45                    │         │    └ typing.Dict[str, typing.Any]
2023-04-24 16:18:45                    │         └ typing.List
2023-04-24 16:18:45                    └ typing.Dict
2023-04-24 16:18:45 
2023-04-24 16:18:45   File "/fides/src/fides/api/ops/task/graph_task.py", line 717, in run_access_request
2023-04-24 16:18:45     v = delayed(get(dsk, TERMINATOR_ADDRESS, num_workers=1))
2023-04-24 16:18:45         │       │   │    └ __TERMINATE__:__TERMINATE__
2023-04-24 16:18:45         │       │   └ {delighted:person: (<bound method GraphTask.access_request of <class 'fides.api.ops.task.graph_task.GraphTask'>:delighted:per...
2023-04-24 16:18:45         │       └ <function get at 0xffff640deb90>
2023-04-24 16:18:45         └ <function delayed at 0xffff640c9480>
2023-04-24 16:18:45 
2023-04-24 16:18:45   File "/root/.local/lib/python3.10/site-packages/dask/threaded.py", line 89, in get
2023-04-24 16:18:45     results = get_async(
2023-04-24 16:18:45               └ <function get_async at 0xffff640789d0>
2023-04-24 16:18:45   File "/root/.local/lib/python3.10/site-packages/dask/local.py", line 511, in get_async
2023-04-24 16:18:45     raise_exception(exc, tb)
2023-04-24 16:18:45     │               │    └ <traceback object at 0xffff542f32c0>
2023-04-24 16:18:45     │               └ ClientUnsuccessfulException("Client call failed with status code '404'")
2023-04-24 16:18:45     └ <function reraise at 0xffff640788b0>
2023-04-24 16:18:45   File "/root/.local/lib/python3.10/site-packages/dask/local.py", line 319, in reraise
2023-04-24 16:18:45     raise exc
2023-04-24 16:18:45           └ ClientUnsuccessfulException("Client call failed with status code '404'")
2023-04-24 16:18:45   File "/root/.local/lib/python3.10/site-packages/dask/local.py", line 224, in execute_task
2023-04-24 16:18:45     result = _execute_task(task, data)
2023-04-24 16:18:45              │             │     └ {__ROOT__:__ROOT__: [{'email': '[REDACTED]@ethyca.com', 'phone_number': ''}]}
2023-04-24 16:18:45              │             └ (<bound method GraphTask.access_request of <class 'fides.api.ops.task.graph_task.GraphTask'>:delighted:survey_response>, __RO...
2023-04-24 16:18:45              └ <function _execute_task at 0xffff64209cf0>
2023-04-24 16:18:45   File "/root/.local/lib/python3.10/site-packages/dask/core.py", line 119, in _execute_task
2023-04-24 16:18:45     return func(*(_execute_task(a, cache) for a in args))
2023-04-24 16:18:45            │      │                │               └ (__ROOT__:__ROOT__,)
2023-04-24 16:18:45            │      │                └ {__ROOT__:__ROOT__: [{'email': '[REDACTED]@ethyca.com', 'phone_number': ''}]}
2023-04-24 16:18:45            │      └ <function _execute_task at 0xffff64209cf0>
2023-04-24 16:18:45            └ <bound method GraphTask.access_request of <class 'fides.api.ops.task.graph_task.GraphTask'>:delighted:survey_response>
2023-04-24 16:18:45 
2023-04-24 16:18:45   File "/fides/src/fides/api/ops/task/graph_task.py", line 154, in result
2023-04-24 16:18:45     raise raised_ex  # type: ignore
2023-04-24 16:18:45           └ ClientUnsuccessfulException("Client call failed with status code '404'")
2023-04-24 16:18:45 
2023-04-24 16:18:45   File "/fides/src/fides/api/ops/task/graph_task.py", line 92, in result
2023-04-24 16:18:45     return func(*args, **kwargs)
2023-04-24 16:18:45            │     │       └ {}
2023-04-24 16:18:45            │     └ (<class 'fides.api.ops.task.graph_task.GraphTask'>:delighted:survey_response, [{'email': '[REDACTED]@ethyca.com', 'phone_number':...
2023-04-24 16:18:45            └ <function GraphTask.access_request at 0xffff5fd11ea0>
2023-04-24 16:18:45 
2023-04-24 16:18:45   File "/fides/src/fides/api/ops/task/graph_task.py", line 525, in access_request
2023-04-24 16:18:45     output: List[Row] = self.connector.retrieve_data(
2023-04-24 16:18:45             │    │      │    │         └ <function SaaSConnector.retrieve_data at 0xffff7bbf6f80>
2023-04-24 16:18:45             │    │      │    └ <fides.api.ops.service.connectors.saas_connector.SaaSConnector object at 0xffff5428e5f0>
2023-04-24 16:18:45             │    │      └ <class 'fides.api.ops.task.graph_task.GraphTask'>:delighted:survey_response
2023-04-24 16:18:45             │    └ typing.Dict[str, typing.Any]
2023-04-24 16:18:45             └ typing.List
2023-04-24 16:18:45 
2023-04-24 16:18:45   File "/fides/src/fides/api/ops/service/connectors/saas_connector.py", line 225, in retrieve_data
2023-04-24 16:18:45     processed_rows, next_request = self.execute_prepared_request(  # type: ignore
2023-04-24 16:18:45                                    │    └ <function SaaSConnector.execute_prepared_request at 0xffff7bbf70a0>
2023-04-24 16:18:45                                    └ <fides.api.ops.service.connectors.saas_connector.SaaSConnector object at 0xffff5428e5f0>
2023-04-24 16:18:45 
2023-04-24 16:18:45   File "/fides/src/fides/api/ops/service/connectors/saas_connector.py", line 279, in execute_prepared_request
2023-04-24 16:18:45     response: Response = client.send(prepared_request, saas_request.ignore_errors)
2023-04-24 16:18:45                          │      │    │                 │            └ False
2023-04-24 16:18:45                          │      │    │                 └ SaaSRequest(request_override=None, path='/v1/survey_responses.json', method='GET', headers=[], query_params=[QueryParam(name=...
2023-04-24 16:18:45                          │      │    └ SaaSRequestParams(method='GET', path='/v1/survey_responses.json', headers={}, query_params={'person_email': '[REDACTED]@ethyca.co...
2023-04-24 16:18:45                          │      └ <function AuthenticatedClient.send at 0xffff7bbf4c10>
2023-04-24 16:18:45                          └ <fides.api.ops.service.connectors.saas.authenticated_client.AuthenticatedClient object at 0xffff54464d60>
2023-04-24 16:18:45 
2023-04-24 16:18:45   File "/fides/src/fides/api/ops/service/connectors/saas/authenticated_client.py", line 140, in result
2023-04-24 16:18:45     raise last_exception  # type: ignore
2023-04-24 16:18:45           └ ClientUnsuccessfulException("Client call failed with status code '404'")
2023-04-24 16:18:45 
2023-04-24 16:18:45 fides.api.ops.common_exceptions.ClientUnsuccessfulException: Client call failed with status code '404'
```
</details> 

I was able to test this out manually using the API calls themselves. A request to `https://api.delighted.com/v1/survey_responses.json?person_email=[email that doesn't exist with no survey responses]` returned a `404`. A request to `https://api.delighted.com/v1/survey_responses.json?person_email=[email that exists with survey responses]`.

#### Steps to Reproduce

1. Spin up a Fides instance
2. Create a Delighted connection with valid credentials
3. Run an access request for a user that has absolutely no data to be retrieved/deleted in Delighted.
4. Approve access request and watch status.

### Expected behavior

Expected to treat the `404` as an "expected error" since there was no data to be retrieved for the nonexistent user in Delighted

### Screenshots

<img width="1043" alt="image" src="https://user-images.githubusercontent.com/39230492/234122234-aead14c9-6bd9-4a0f-b7ab-a678b5ec15ed.png">
<img width="364" alt="image" src="https://user-images.githubusercontent.com/39230492/234122268-d7c7df8c-5df3-42ca-a6bb-13901f96a0b8.png">

### Environment

* Version: built from main, tested on 2.11
* OS:
* Python Version:
* Docker Version:

### Additional context

_Add any other context about the problem here._

**Comment by RobertKeyser** [comment]
Originally, I was going to solve this by adding in `ignore_errors: true` to the saas_config. 

```yaml
    - name: survey_response
      requests:
        read:
          method: GET
          ignore_errors: true # When no record is found for the provided email, Delighted returns a 404.
          path: /v1/survey_responses.json
          query_params:
            - name: person_email
              value: <email>
          param_values:
            - name: email
              identity: email
```

However, in doing some more research on this, I don't think that there will ever be a case where `person_email` is not ALSO an email tied to a person. According to the Delighted API docs, when you delete a person, you also delete the following:

> Surveys (including any scheduled or pending surveys)
> Responses (scores, comments, notes, and tags)
> Properties
> Autopilot membership
> Survey history
> Links to integrations (such as Segment or Intercom)
> Unsubscribe/bounce status
> Testimonials

With that, I think `person_email` essentially points to `person.email` like a foreign key. Instead of ignoring errors on this endpoint, we can instead rearchitect it so it's a dependent endpoint of `person`.

**Comment by RobertKeyser** [comment]
> Originally, I was going to solve this by adding in `ignore_errors: true` to the saas_config.
> 
> ```yaml
>     - name: survey_response
>       requests:
>         read:
>           method: GET
>           ignore_errors: true # When no record is found for the provided email, Delighted returns a 404.
>           path: /v1/survey_responses.json
>           query_params:
>             - name: person_email
>               value: <email>
>           param_values:
>             - name: email
>               identity: email
> ```
> 
> However, in doing some more research on this, I don't think that there will ever be a case where `person_email` is not ALSO an email tied to a person. According to the Delighted API docs, when you delete a person, you also delete the following:
> 
> > Surveys (including any scheduled or pending surveys)
> > Responses (scores, comments, notes, and tags)
> > Properties
> > Autopilot membership
> > Survey history
> > Links to integrations (such as Segment or Intercom)
> > Unsubscribe/bounce status
> > Testimonials
> 
> With that, I think `person_email` essentially points to `person.email` like a foreign key. Instead of ignoring errors on this endpoint, we can instead rearchitect it so it's a dependent endpoint of `person`.

Is that as simple as the following?

```yaml
    - name: survey_response
      requests:
        read:
          method: GET
          path: /v1/survey_responses.json
          query_params:
            - name: person_email
              value: <email>
          param_values:
            - name: email
              references:
                - dataset: <instance_fides_key>
                  field: person.email
                  direction: from
```

**Comment by RobertKeyser** [comment]
Likely to be released in `2.14.0`
